utils.py

```diff
@@ -1,8 +1,32 @@
 from typing import Iterable, Optional, Dict, Sequence
 
+import jinja2
 import requests
 
 
+def traverse_into(name, **namespace):
+    head = namespace[name[0]]
+    for path in name[1:]:
+        if isinstance(head, dict):
+            head = head.get(path, None)
```

Comment:
Consider to directly access the part so that an exception is triggered.

```suggestion
            head = head[path]
```
This probably then needs explicit handling at another point (one where you know if this is actually an error or not).